compiler/daml-lf-tools/src/DA/Daml/LF/Simplifier.hs

```diff
@@ -123,6 +123,20 @@ safetyStep = \case
       BEMulDecimal        -> Safe 1
       BEDivDecimal        -> Safe 1
       BERoundDecimal      -> Safe 1
+      BEEqualNumeric      -> Safe 2
+      BELessNumeric       -> Safe 2
+      BELessEqNumeric     -> Safe 2
+      BEGreaterNumeric    -> Safe 2
+      BEGreaterEqNumeric  -> Safe 2
+      BEAddNumeric        -> Safe 1
+      BESubNumeric        -> Safe 1
+      BEMulNumeric        -> Safe 1
+      BEDivNumeric        -> Safe 1
+      BEInt64ToNumeric    -> Safe 0
+      BENumericToInt64    -> Safe 0
+      BENumericFromText   -> Safe 1
+      BEToTextNumeric     -> Safe 1
+      BERoundNumeric      -> Safe 1
```

Comment:
@hurryabit Just to confirm, type applications aren't counted here, right? Otherwise these numbers would have to be higher. (And so would the numbers for BEFoldl, BEFoldr, etc.) But then `BEMapEmpty` says `Safe 1` but it should be `Safe 0`?

Reply:
Correct, type applications are not counted. (See the cases for `ETyApp` and `ETyLam` below.) The `Safe 1` for `BEMapEmpty` is wrong and must have slipped through my code review. I'll fix it in a separate PR.